### PR TITLE
Apim 3567 doc page badge

### PR DESCRIPTION
**Pull request by loriepisicchio**
## Issue

https://gravitee.atlassian.net/browse/APIM-3567

## Description

Add badge next to page name to show the page type

Screenshot 2024-02-06 at 09 51 50

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-zkyqbhklzg.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.ts [hunk 1]
```diff
@@ -15,9 +15,10 @@
  */
 
 import { Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges } from '@angular/core';
-import { MatLegacyTableDataSource as MatTableDataSource } from '@angular/material/legacy-table';
+import { MatTableDataSource } from '@angular/material/table';
```
Comment (jourdiw) [inline gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.ts:18]: Why did you change the source of the mat table?

Comment (loriepisicchio) [reply gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.ts:18]: I changed to whole mat-table component from legacy to new module.

**Review by jourdiw — APPROVED** [review]
